.github/workflows/test.yml

```diff
@@ -487,6 +487,20 @@ jobs:
       parallelism: 1 # because single GPU node
       post_summary: false # Will post consolidated summary instead
 
+  nebius-multi-gpu:
```

Comment:
No need to create a new job here. Let's simply use the single Nebius job. Multiple GPUs or not does not need to be specified from the CI. The tests are configured to be skipped automatically if the number of GPUs in the spec is not available.